src/command/pod/list.ts

```diff
@@ -0,0 +1,22 @@
+import { LeafCommand } from 'furious-commander'
+import { createKeyValue } from '../../utils/text'
+import { PodCommand } from './pod-command'
+
+export class List extends PodCommand implements LeafCommand {
+  public readonly name = 'list'
+
+  public readonly alias = 'ls'
+
+  public readonly description = 'List pods'
+
+  public async run(): Promise<void> {
+    await super.init()
+
+    await this.fillFdpAccount(this.account, this.password)
+    const pods = await this.fdpStorage.personalStorage.list()
+    for (const pod of [...pods.getPods(), ...pods.getSharedPods()]) {
```

Comment:
I think it could have 2 lists instead or at least notating whether the pod is shared or not. It is important information since the shared pod won't be writeable.

Reply:
it is still a todo IMO